### PR TITLE
Improvements to standalone scripts

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Small improvements as a follow up to #11098

In #13841 I introduced the batch size argument for the standalone script, which worked fine for the CI but unfortunately it also broke the following use case, which is handy for locally debugging:

```
bash run_standalone_tests.sh -k <the_name_of_the_test_case_to_run>
```

I'm reverting this back to an environment variable because the argument parsing in bash would otherwise get a bit too nasty. 

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
I made sure I had fun coding 🙃

cc @carmocca @akihironitta @borda @justusschock @awaelchli @rohitgr7

### CODE REVIEW

File: tests/tests_pytorch/run_standalone_tests.sh [hunk 1]
```diff
@@ -80,31 +80,7 @@ done
...
-fi
-report+="Ran\tplugins/environments/torch_elastic_deadlock.py\n"
-
-# test that a user can manually launch individual processes
-export PYTHONPATH="${PYTHONPATH}:$(pwd)"
-args="--trainer.accelerator gpu --trainer.devices 2 --trainer.strategy ddp --trainer.max_epochs=1 --trainer.limit_train_batches=1 --trainer.limit_val_batches=1 --trainer.limit_test_batches=1"
-MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=1 python ../../examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args} &
-MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=0 python ../../examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args}
-report+="Ran\tmanual ddp launch test\n"
+echo "Batched mode finished. End of standalone tests."
```
Comment (carmocca) [inline tests/tests_pytorch/run_standalone_tests.sh:83]: We can remove this echo now that there are not further steps

```suggestion
```

File: tests/tests_pytorch/run_standalone_tasks.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+report+="Ran\tplugins/environments/torch_elastic_deadlock.py\n"
+
+# test that a user can manually launch individual processes
+export PYTHONPATH="${PYTHONPATH}:$(pwd)"
+args="--trainer.accelerator gpu --trainer.devices 2 --trainer.strategy ddp --trainer.max_epochs=1 --trainer.limit_train_batches=1 --trainer.limit_val_batches=1 --trainer.limit_test_batches=1"
+MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=1 python ../../examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args} &
+MASTER_ADDR="localhost" MASTER_PORT=1234 LOCAL_RANK=0 python ../../examples/convert_from_pt_to_pl/image_classifier_5_lightning_datamodule.py ${args}
+report+="Ran\tmanual ddp launch test\n"
+
+# echo test report
```
Comment (carmocca) [inline tests/tests_pytorch/run_standalone_tasks.sh:44]: Slightly leaning towards removing the report from this file

**Review by carmocca — APPROVED** [review]

**Comment by carmocca** [comment]
We use the `refactor` label for code refactors (this is just a CI change) and the `tests` label for direct changes to the tests (again, this is just a CI change). Removed them

File: tests/tests_pytorch/run_standalone_tasks.sh (hunk 2 above)
Comment (awaelchli) [reply tests/tests_pytorch/run_standalone_tasks.sh]: updated

**Comment by carmocca** [comment]
#11098 was merged before, but I would still merge the small improvements here.

**Review by kaushikb11 — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Pull request merged**